### ISSUE TITLE
GobblinYarnAppLauncher tries to send email even if email notification is disabled

### ISSUE THREAD

**Issue by kadaan** [linkedin/gobblin#712]
```
Exception in thread "Thread-4" java.lang.NullPointerException
        at javax.mail.internet.InternetAddress.parse(InternetAddress.java:678)
        at javax.mail.internet.InternetAddress.parse(InternetAddress.java:638)
        at javax.mail.internet.InternetAddress.<init>(InternetAddress.java:111)
        at org.apache.commons.mail.Email.createInternetAddress(Email.java:1895)
        at org.apache.commons.mail.Email.setFrom(Email.java:749)
        at org.apache.commons.mail.Email.setFrom(Email.java:732)
        at org.apache.commons.mail.Email.setFrom(Email.java:712)
        at gobblin.util.EmailUtils.sendEmail(EmailUtils.java:54)
        at gobblin.yarn.GobblinYarnAppLauncher.sendEmailOnShutdown(GobblinYarnAppLauncher.java:749)
        at gobblin.yarn.GobblinYarnAppLauncher.access$400(GobblinYarnAppLauncher.java:130)
        at gobblin.yarn.GobblinYarnAppLauncher$2.run(GobblinYarnAppLauncher.java:769)
```

**Comment by kadaan** [comment]
I'm claiming this.

**Comment by kadaan** [comment]
Fixed by #713
